### PR TITLE
[DynamoDb] Prepare for 0.3.1

### LINKED ISSUE #695: Release DynamoDB
Hello !

Any chance to have a new release of DynamoDB including my latest change about nullable array ?

Thanks !

### PR DESCRIPTION
**Pull request by Nyholm**
This will fix #695

### CODE REVIEW

File: src/Service/DynamoDb/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 ## NOT RELEASED
 
+## 0.3.1
+
+### Added
+
+- BatchWriteItem
+- BatchGetItem
```
Comment (jderusse) [inline src/Service/DynamoDb/CHANGELOG.md:10]:
```suggestion
- Added operation `BatchWriteItem`
- Added operation `BatchGetItem`
```

Comment (Nyholm) [reply src/Service/DynamoDb/CHANGELOG.md]: Thank you

**Comment by Nyholm** [comment]
Thank you

**Comment by tyx** [comment]
Hi ! in fact I did not see yet this release : https://github.com/async-aws/dynamo-db/releases :smile:

**Comment by Nyholm** [comment]
Oops. I must have forgotten. 

I saw that @jderusse fixed it just now https://github.com/async-aws/dynamo-db/releases/tag/0.3.1

**Pull request merged**